src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Conditions/Condition.cs

```diff
@@ -15,29 +15,29 @@ public enum Condition
     {
         [DisplayName("Contains")]
         Contains,
-        [DisplayName("StartsWith")]
```

Comment:
Not sure if this should be changed. This was using the resources inside the resx files and was acting as a key for the resource.